dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -0,0 +1,903 @@
+<template>
+  <Card class="col-12 page-wrapper-card">
+    <template #title>New Dataset - LkSG </template>
+    <template #content>
+      <div class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
+            v-model="lkSGDataModel"
+            :actions="false"
+            type="form"
+            id="createLkSGForm"
```

Comment:
Are deviations of the questions here caused by a deviation between the figma and the excel file?

Reply:
And not all fields were designed by the UX/UI team?

Reply:
I don't know about that, since Marcin implemented this component.

Reply:
We try to reflect excel and data model not figma

Reply:
Ok, then it is fine